views/partials/site-header-default.twig

```diff
@@ -0,0 +1,7 @@
+<header class="site-header">
```

Comment:
Shouldn't the sticky header be applied to the default header also? See if we can reduce the duplicate markup also between the two, so only the inner content has the default fallback; everything else is shared.